pkg/log/default.go

```diff
@@ -0,0 +1,31 @@
+package log
+
+import "github.com/sirupsen/logrus"
+
+var defaultPatterns = []MaskPattern{
+	// JWT
+	NewRegexMaskPattern(`eyJ[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*`),
+	// Session Tokens
+	NewRegexMaskPattern(`[A-Fa-f0-9-]{36}\.[A-Za-z0-9]+`),
+}
+
+func NewDefaultLogHook(sensitiveStrings []string) logrus.Hook {
```

Comment:
This hook is for masking potentially sensitive strings. Maybe rename it to `NewMaskingLogHook`?